### ISSUE TITLE
Chromium freezes at start

### ISSUE THREAD

**Issue by ambrop72** [NixOS/nixpkgs#26730]
## Issue description

With the current release-17.03 channel after the latest Chromium updates (58.0.3029.110), Chromium no longer works in NixOS, it starts up seemingly fine but freezes after about a second.

## Technical details

nixpkgs channels/nixos-17.03 76d649b59484607901f0c1b8f737d8376a904019

**Comment by bjornfor** [comment]
I run ```/nix/store/7x3rzc9aynqn49vnh3wg3cr77m31hzqd-chromium-58.0.3029.110/bin/chromium``` on NixOS. No crash. (I have it running for 24+ hours.)

**Comment by ambrop72** [comment]
@bjornfor What I have is `/nix/store/vvj0pmp3gjiaha95fxig4gknavn6fs47-chromium-58.0.3029.110` which corresponds to the latest build of 17.03: https://hydra.nixos.org/build/54720449

**Comment by bjornfor** [comment]
Hm, maybe I built from git? (It seems I have the same chromium version, but different nix store path.)

**Comment by ambrop72** [comment]
I suspect that commit "chromium: apply patch to fix chromecast" broke it because I have a working Chromium previously built based on cherry-picking from master, and its chromium expressions are the same 
 as currently is in release-17.03 except for the lack of that patch.

**Comment by ambrop72** [comment]
Wrong guess, reverting that patch does not fix anything. Some dependency must be causing the issue.

**Comment by ambrop72** [comment]
Found the culprit by bisecting: https://github.com/NixOS/nixpkgs/commit/d33c4b2b4ab2f664ba9710cabcd1178cd3cac882

It must be nss or nspr updates that broke it.

Note that to be able to do this bisect, I rebased channels/nixos-17.03 on top of my branch where I had previously cherry-picked chromium updates. This is the branch where the bisect was actually done: https://github.com/ambrop72/nixpkgs/tree/chromium-bisect (in this case the breaking commit is 7248d1a97b6b001e44dc449513cd899ec8fee4bb).
